src/Config.ts

```diff
@@ -9,9 +9,10 @@ export interface Config {
 
     /**
      * Name of the type/interface to generate schema for.
+     * Can specify more than once to generate multiple schemas.
```

Comment:
```suggestion
    * Name of the type(s)/interface(s) to generate schema for.
```

it's wrong that it can be specified more than once

Reply:
Hmm I'd have thought it's right as is. The way I have it wired up, for the CLI, you have to do `--type foo --type bar`. You can't do `--type foo bar`. Eg:

```
$ npm run --silent run -- \
      --path ~/<snip> \
      --type Foo \
      --type Bar \
      --tsconfig ~/<snip>/tsconfig.json \
      --no-type-check >/dev/null

$ npm run --silent run -- \
      --path ~/<snip> \
      --type Foo Bar \
      --tsconfig ~/<snip>/tsconfig.json \
      --no-type-check >two.txt
error: too many arguments. Expected 0 arguments but got 1.
```

Do you still want the suggested change?

Reply:
I thought it was odd to say this on the ts interface. You cannot use the same key twice.

Reply:
Ah I see. I had been thinking of this as CLI for some reason. Updated to:

```
Name of the type(s)/interface(s) to generate schema(s) for.
```

Reply:
Whoops nevermind, I see that you applied the edit. Ignore me 🫡